### PR TITLE
Add attach file button to chat input

### PR DESCRIPTION
**Pull request by AbhisekOmkar**
Introduce an SVG-based attach file button to allow users to add attachments to their messages.
- Adjust input field and send button styling to accommodate the new button.
- Enhance user interaction by providing a clear visual cue for attaching files.
Screenshot 2024-03-16 at 10 54 43 PM

### CODE REVIEW

**Review by neubig — CHANGES_REQUESTED** [review]
Thanks for sending this in!

This PR is also doing some formatting that is independent of the main change though, so it's a bit hard for me to review.

Could you please limit the changes to just be related to the main topic of the PR (and possibly send a separate PR for formatting)? Thanks!

**Comment by AbhisekOmkar** [comment]
Sure, I understand your request. I will make sure to limit the changes in the PR to only those related to the main topic and avoid including formatting changes. Thank you for bringing this to my attention, and I'll ensure the PR focuses solely on the intended modifications.

**Comment by AbhisekOmkar** [comment]
@huybery can you merge it

**Comment by JustinLin610** [comment]
> @huybery can you merge it

Sry, I still need @neubig Graham to take a final look. Me and @huybery do not have expertise in this field. Apparently I think you have fixed the formatting issue but I am not so sure about this.

**Comment by AbhisekOmkar** [comment]
> > @huybery can you merge it
> 
> Sry, I still need @neubig Graham to take a final look. Me and @huybery do not have expertise in this field. Apparently I think you have fixed the formatting issue but I am not so sure about this.

I have checked the this on my local machine its working as expected and i have attached screenshot as well.

**Comment by neubig** [comment]
I'll take a look but might not have time till tomorrow!

**Comment by neubig** [comment]
Hi @AbhisekOmkar  , thanks so much again for the patience!

We just [merged in some linting/formatting stuff](https://github.com/OpenDevin/OpenDevin/pull/32) that will standardize our formatting and make this PR easier to review. Could you please `git pull origin main` to merge in these changes, fix any merge conflicts, and I'll review after that?

**Comment by AbhisekOmkar** [comment]
@neubig Its done now

File: frontend/package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@heroicons/react": "^2.1.1",
```
Comment (neubig) [inline frontend/package.json:6]: Is this dependency being used? I could see it used anywhere.

**Review by neubig — CHANGES_REQUESTED** [review]
Looks good, but it seems that there's an unused dependency added (unless I'm mistaken). It'd be good if you could remove that!

EDIT: Also linting is failing, please resolve that.

File: frontend/package.json (hunk 1 above)
Comment (AbhisekOmkar) [reply frontend/package.json:6]: its not being used now i will remove it

**Comment by AbhisekOmkar** [comment]
Im closing the PR seems to be some issues in the local machine

**Pull request closed without merging**